docs/glues/suma2text.md

```diff
@@ -1,137 +1,11 @@
 ---
 id: suma2text
-title: Error to Text - Suma2Text
 sidebar_label: Error to Text
 slug: /glues/suma2text
+hide_title: true
```

Comment:
Why not create an `.md` generator? we could have all this information inside 1 file in json format, so the generator generates everything before deploying

Another suggestion is, why not read all repositories so auto-creates these files? it looks simple as all files have a similar structure